Source/Raymarcher/Shaders/Private/WindowedRaymarchMaterials.usf

```diff
@@ -135,47 +135,106 @@ float4 PerformWindowedRaymarchOctree(Texture3D DataVolume, // Data Volume
 	// Copy the octree depth on mip level 0 for use when recalculating to correct texture coordinates.
 	float OctreeDepthConst = OctreeDepth;
 
-	// Get the size of the volume on requested mip. Used to recalculate coordinates to correct format.
-	OctreeVolume.GetDimensions(OctreeMip, OctreeWidth, OctreeHeight, OctreeDepth, OctreeMipCount);
+	// Get the bit number based on window center and window width.
+	uint WindowNumber = GetWindowingParamsBitNumber(WindowingParams.x, WindowingParams.y);
 
-	// Values from the previous iteration.
-    for (int i = 0; i < MaxSteps; i++)
-    {
-        CurPos += LocalCamVec;
-    	// Because we jitter only "against" the direction of LocalCamVec, start marching before first sample.
-	    // Any position that is clipped by the clipping plane shall be ignored.
-        if (!IsCurPosClipped(CurPos, ClippingCenter, ClippingDirection))
-        {
-        	// Calculate the correct position in octree. The Z coordinate needs to be multiplied by ratio of the base volume depth vs base octree volume depth.
-        	// Multiply all the values by their respective volume size to get actual texel coordinates instead af UV coordinates.
-        	int3 VoxelPos = float3(CurPos.x * OctreeWidth, CurPos.y * OctreeHeight, (CurPos.z * DataVolumeDepth / OctreeDepthConst) * OctreeDepth);
+	// Set to the mip number with worst detail.
+	const int MaxPossibleMip = 3;
+	int CurrentMip = OctreeMip;
 
-        	float4 ColorSample = SampleWindowedVolumeOctreeStep(VoxelPos, StepSizeWorld, OctreeVolume,
-                                               TF, Material.Clamp_WorldGroupSettings, WindowingParams, OctreeMip);
+	// Check if the entry position should be visible over plane. 
+	if(IsCurPosClipped(CurPos, ClippingCenter, ClippingDirection))
```

Comment:
Remove this from here. This will be implemented properly in `PerformRaymarchCubeSetup`, where entry/exit positions will be modified depending on the clip plane for all raymarch types.